packages/components/collapse/src/collapse-item.vue

```diff
@@ -13,12 +13,16 @@
       @focus="handleFocus"
       @blur="focusing = false"
     >
-      <slot name="title">{{ title }}</slot>
-      <slot name="icon" :is-active="isActive">
-        <el-icon :class="arrowKls">
-          <component :is="icon" />
-        </el-icon>
-      </slot>
+      <div :class="itemTitleKls">
+        <slot name="title">{{ title }}</slot>
+      </div>
+      <div :class="expandKls">
+        <slot name="icon" :is-active="isActive">
+          <el-icon :class="arrowIconKls">
+            <component :is="icon" />
+          </el-icon>
+        </slot>
+      </div>
```

Comment:
It is best not to modify the structure here. I think it is more elegant to meet the needs through style.